src/pages/search/registration_search/ReportsPage.tsx

```diff
@@ -0,0 +1,58 @@
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { Box, Button, IconButton, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import reportThumbnail from '../../../resources/images/report-thumbnail.png';
+
+const ReportsPage = () => {
+  const { t } = useTranslation();
+
+  const [selectedReport, setSelectdReport] = useState('');
+
+  return selectedReport ? (
+    <Box>
+      <IconButton title={t('common.back')} onClick={() => setSelectdReport('')}>
+        <ArrowBackIcon />
+      </IconButton>
+      <iframe
```

Comment:
kunne nok med fordel vært litt smalere på mobil? litt usikker på hvor fint vi får det når det er iframe, men ser for meg det hadde vært bedre om det ikke hadde vært bredere enn vår side. Selv om det vil bety enda mer horisontal scroll da
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/08002752-5914-4a18-b383-7fa17550160a)

Reply:
Sikkert ikke mulig, siden vi må vise hele iframen uansett